src/Time/Time.cpp

```diff
@@ -135,3 +314,16 @@ size_t hash<Time>::operator()(const Time& t) const noexcept {
   return boost::hash<Time>{}(t);
 }
 }  // namespace std
+
+// Miscellaneous other functions for TimeDelta
+
+TimeDelta abs(TimeDelta t) noexcept {
+  if (not t.is_positive()) {
+    t *= -1;
```

Comment:
is this standard behavior to modify the argument in-place and also return it? Should there be a non-modifying version?

Reply:
`t` isn't a reference.

Reply:
oh right, nvm